src/components/MoreInformation.js

```diff
@@ -5,60 +5,119 @@ import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
 import { makeStyles } from "@material-ui/core";
 
 const useStyles = makeStyles((theme) => ({
-    accordion: {
-        "margin-top": theme.spacing(2),
-        "margin-bottom": theme.spacing(2),
-        ":last-child": { "margin-bottom": theme.spacing(2) },
-    },
-    buttonText: {
-        ...theme.typography.button,
-    },
-    extraDataContainer: {
-        display: "block",
-    },
-    extraData: {
-        display: "block",
-        "padding-bottom": theme.spacing(1),
-    },
+  accordion: {
+    "margin-top": theme.spacing(2),
+    "margin-bottom": theme.spacing(2),
+    ":last-child": { "margin-bottom": theme.spacing(2) },
+  },
+  buttonText: {
+    ...theme.typography.button,
+  },
+  extraDataContainer: {
+    display: "block",
+  },
+  extraData: {
+    display: "block",
+    "padding-bottom": theme.spacing(1),
+  },
+  formattedExtraData: {
+    color: "#F50057",
+  },
 }));
 
 export default function MoreInformation({ entry }) {
-    const classes = useStyles();
-    return (
-        <Accordion className={classes.accordion}>
-            <AccordionSummary
-                expandIcon={<ExpandMoreIcon />}
-                className={classes.buttonText}
-            >
-                More Information
-            </AccordionSummary>
-            <AccordionDetails className={classes.extraDataContainer}>
-                <div className={classes.extraData}>
-                    <b>Address:</b> {entry.streetAddress}, {entry.city}, MA{" "}
-                    {entry.zip}
-                </div>
-                <ExtraData data={entry.extraData} />
-            </AccordionDetails>
-        </Accordion>
-    );
+  const classes = useStyles();
+  if (!(entry.extraData == null)) {
+    //reggie lewis uses markdown but not for emphasis so it skips that site
+    if (!(entry.location == "Reggie Lewis State Track Athletic Ctr")) {
+      var currentData = entry.extraData["Additional Information"];
```

Comment:
You should go through and change all of these instances of `var` to either `const` or `let`.
The difference:
`const` is if you never reassign the variable. For example, `currentData` is never reassigned so it can be a `const`.
`let` allows reassignment. Gotcha: if you are setting object properties, the object can be a const. For example,
```
const objectExample = {};
objectExample.key = "value";
```
is legal and appropriate (imo).